src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+const ADD = 'ADD_BOOK';
+const REMOVE = 'REMOVE_BOOK';
+
+export const addBook = ({ name, category, author }) => ({
+  type: ADD,
+  name,
+  category,
+  author,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE,
+  id,
+});
```

Comment:
- [ ] Your app is not adding or removing any new book. When I click on the button remove it does not remove it and when I try to type something in the input it doesn't allow me to type something.